feat(service-version): adjust docs [tdx-2281] (#4334)
Kong/docs.konghq.com: app/konnect/servicehub/service-versions.md

Instead, create a new version of the service in the new environment when you're
ready to move to it.

Every service version may be in one of the states: "Published", "Deprecated" and "Unpublished":
* **Published**: This indicates that the service version is ready to be shared with the API consumers. It is exposed to Dev Portal where developers can request access to consume it via the {{site.base_gateway}}. This is default state.

> * **Deprecated**: This indicates that the service version will be deprecated soon. It will still be exposed to the Dev Portal and receive API request via {{site.base_gateway}}. Banner with information about deprecation will be displayed at the top of the service version page in Dev Portal.

Reviewer: I think we should also indicate that Dev Portal users with applications registered to this service version will receive an email notification that this version has been deprecated.

1. Click **Create** to save.

## Managing a service version lifecycle

Changing service version status will have an impact on its visibility in Dev Portal. 

From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service, then selects its service version which status you would like to adjust. Then follow these steps:

> 1. From the **Service version actions** drop-down menu, select **Edit version status**.

Reviewer 2: ```suggestion
1. Click  **Service version actions** > **Edit version status**.
```

> 2. In dialog you can select appropriate status. Next to each status there is a brief explanation which pops up on icon hover.

Reviewer 2: ```suggestion
1. Select a service version status.
```

>     1. In some cases there might be an additional step to fulfill.

Reviewer 2: @tomaszwylezek What other steps would there be here?

Author: In the case of changing the version status into "deprecated" - we have one more step in which the user is displayed information about that there will be - check video here - https://github.com/Kong/khcp-ui/pull/2642

> 3. Click **Save** to adjust status of service version.

Reviewer 2: ```suggestion
1. Click **Save**.
```

## Delete a service version

Deleting a service version permanently removes it and its implementation, routes, and plugins from the Service Hub.

> From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service version, then delete it:

Reviewer 2: ```suggestion
1. From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select the service version you want to delete.
```

> * From the **Service version actions** drop-down menu, select **Delete service version**, then confirm deletion in the dialog.

Reviewer 2: ```suggestion
1. Click **Service version actions** > **Delete service version**.
```

> * Confirm deletion by typing name of service version and click **Yes, delete**

Reviewer 2: ```suggestion
1. Confirm the deletion by typing the name of the service version and clicking **Yes, delete**.
```

1. Click **Create** to save.

## Manage the service version lifecycle

The service version lifecycle determines how and if a service version is displayed in the Dev Portal. 

1. From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), choose the service you want to manage the lifecycle of.

2. Select the service version you want to manage.

3. Click  **Service version actions** > **Edit version status**.

4. Select a service version status.
   

>     1. In some cases there might be an additional step to fulfill.

Reviewer 2: ```suggestion
    1. Deprecation only: Click **Save** on the dialog to deprecate the service version and send consumers an email notification.
```
